apparmor.d/profiles-m-r/mumble

```diff
@@ -56,6 +58,9 @@ profile mumble @{exec_path} {
        owner @{PROC}/@{pid}/mounts r,
   deny       @{PROC}/sys/kernel/random/boot_id r,
 
+  @{lib}/ r,
+  /usr/lib64/ r,
```

Comment:
Does it fail without these rules? It's probably a regular exploration by some library, which is not necessarily needed for mumble to function.